movie-blog/src/components/create-element.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+class createElement extends React.Component {
```

Comment:
There exists the naming convention to name your components, using upper camel-case. So, `CreateElement` would be expected here.

Reply:
And what about file naming? should they be in `PascalCase` or it isn't important?